firmware/sys/system_init/system_init.c

```diff
@@ -0,0 +1,136 @@
+#include "stdio.h"
+#include "ds18_sensor.h"
+#include "moisture_sensor.h"
+#include "xtimer.h"
+#include "serialization.h"
+#include "udp_m4a.h"
+#include "board.h"
+#include "thread.h"
+#include "esp_system.h"
+
+#include "net/gnrc/netif.h"
+#include "net/netif.h"
+
+#define SERVER_BUFFER_SIZE      (100)
+
+int ds18_is_avaliable = 1;
+int moisture_sensor_is_avaliable = 1;
+uint8_t buffer[SERVER_BUFFER_SIZE];
+int16_t temp = 0;
+int soil_moisture = 0;
+sensor_data payload; 
+int port = 3000;
+char address[30] = "::1";
+char get_sensor_event[THREAD_STACKSIZE_DEFAULT];
+
+#define SLIP_IFACE (10)
```

Comment:
handle with care, this can change from board to board, I mean isn't a dynamic mapping better?